plugins/commands/create.js

```diff
@@ -94,14 +128,18 @@ module.exports = () => ({
             const data = request.payload;
             let commandSpec;
             let commandBin;
+            let multipartCheckResult = { valid: false };
 
             // if Content-type is multipart/form-data, both command binary and meta are posted
```

Comment:
You should better to fix comment because Habitat local mode also posts with Content-type is multipart/form-data.